### ISSUE TITLE
["Bug"] Missing command scr_usekfont

### ISSUE THREAD

**Issue by NightFright2k19** [Shpoike/Quakespasm#68]
When using id1\pak0.pak from Quake Enhanced, there is one command in its embedded quake.rc which QSS currently does not understand: _**scr_usekfont**_. 

While it's probably harmless and a rather cosmetic thing (apparently enables Unicode font rendering in Kex, can be set by mods aware of localization), maybe this warning can at least be suppressed during startup (in order to avoid confusion about something not working properly).

**Comment by adelpha** [comment]
Is QSS just locked to English at the moment? I know it doesn't support UTF-8 which is what the localization files are encoded in so it's just interpreting them as ANSI.

It'sa kinda cool that id remade the Quake font with all the UTF characters required for Russian, French, etc.

**Comment by NightFright2k19** [comment]
AFAIK there is no way to switch language right now, also since QSS doesn't have a menu option for it. You could rename one of the other language files to the name of the English files, but many characters probably won't be displayed. 

Could just as well request fully functional language switching, but I dunno if that is out of scope for this project.

**Comment by adelpha** [comment]
Yeah I did that, QSS interprets the UTF-8 as Quake's ANSI/Extended ASCII and displays characters from that character set instead, i.e. illegible garbage. English works because UTF-8 is backward compatible with ASCII, and DoMA doesn't use any of the Quake extended charset....

...and that's because the traditional Quake extended characters simply don't exist in the remaster when you turn on Unicode support, hence why you don't see any of the red or graphical characters in DoMA at all. I think any mod that includes multi-language support has to give up use of the graphical characters, since it seems to be an all-or-nothing switch.

Here's the confont.png file from QuakeEX.kpf:

![confont](https://user-images.githubusercontent.com/13021896/132742785-7039a8cb-36b0-41f0-bbb8-ae6d0593b290.png)

and the contents of confont.kfont that defines the characters (they're variable sizes instead of the old grid-based layout so this is now necessary):

> texture "fonts/confont.png"
unicode
mapchar
{
	64 1 1 8 12 0
	97 11 1 8 12 0
	99 1 15 8 12 0
	100 11 15 8 12 0
	101 21 1 8 12 0
	102 21 15 8 12 0
	103 1 29 8 12 0
	104 11 29 9 12 0
	105 22 29 4 12 0
	106 28 29 6 12 0
	107 36 1 8 12 0
	108 31 15 8 12 0
	109 36 29 9 12 0
[snipped for brevity]
	8217 50 99 5 12 0
}

**Comment by NightFright2k19** [comment]
I don't really think this could become a thing for any future mods, unless they are coming from Nightdive themselves, maybe. It's still an interesting concept though and worth being supported for the already existing Quake Enhanced content alone.
